src/dune_engine/dep.ml

```diff
@@ -77,15 +77,15 @@ module Fact = struct
     type t =
       { files : Digest.t Path.Map.t
       ; dirs : Digest.t Path.Map.t (* Only for file selectors for now *)
-      ; parent_dirs : Path.Set.t
+      ; build_parent_dirs : Path.Build.Set.t
```

Comment:
I find it hard to grok what this name means. I'd use something more explicit like

```suggestion
      ; dirs_that_must_exist : Path.Build.Set.t
```

and something similar in the `mli`.

Reply:
(Maybe there is an even more descriptive name.)

Reply:
The current name is especially confusing, since it's not only about parent dirs, right?

Reply:
I used your suggested name.